actionpack/lib/action_controller/api.rb

```diff
@@ -128,13 +128,13 @@ def self.without_modules(*modules)
       # Append rescue at the bottom to wrap as much as possible.
       Rescue,
 
-      # Add instrumentations hooks at the bottom, to ensure they instrument
-      # all the methods properly.
-      Instrumentation,
-
       # Params wrapper should come before instrumentation so they are
       # properly showed in logs
```

Comment:
If this comment is accurate, then swapping these middleware will change how parameters are logged.

Could you check what effect this will have, and write a test that describes the new behaviour?